pkg/app/web/src/pages/settings/piped/index.tsx

```diff
@@ -74,6 +74,9 @@ const selectFilteredPipeds = createSelector<
   }
 );
 
+const OLD_KEY_ALERT_MESSAGE =
+  "The old key is still there.\nTo add a new key, you need to delete the old one first.";
```

Comment:
"The old key is still there.\nDo not forget to delete it once you update your Piped to use this new key."